packages/app/src/composables/useSpecFilter.ts

```diff
@@ -25,6 +25,7 @@ export function useSpecFilter (savedFilter?: string) {
 
   function setSpecFilter (specFilter: string) {
     if (specStore.specFilter !== specFilter) {
+      specStore.setSpecFilter(specFilter)
```

Comment:
I deleted this, my b... not the worst, now we know we were missing test coverage.